config/config.go

```diff
@@ -18,12 +18,19 @@ func OptionalEnv(varName string, optional string) string {
 }
 
 /*
-	Check for an environment variable value, exit program if not found
+	Check for an environment variable value or the equivalent docker secret, exit program if not found
```

Comment:
I would prefer not to mention and link the service to a specific technology. For example, the container and your idea are also compatible with Kubernetes.
Could you change it to something like "value or a file".